bench/workgen/setup.py

```diff
@@ -61,7 +61,7 @@
 
 setup(name='workgen', version=wt_ver,
     ext_modules=[Extension('_workgen',
-                [os.path.join(dir, 'workgen_wrap.cxx')],
+                [os.path.join(dir, 'workgen_wrap.cpp')],
```

Comment:
This `workgen_wrap.cpp` file is mentioned in a few places but as far as I can tell doesn't exist. @ddanderson

Reply:
I don't think we do support autotools anymore but I'm still not clear on what this file odes. It's obvious that something isn't correct here but I'm not sure if it should be removed.

Reply:
This used to be used to build the _workgen.so library.  There is also an equivalent one for lang/python, generating _wiredtiger.so.  Python wants to control how "native" libraries are built, and that is done via setup.py.  The developer, or SWIG, puts code that allows calls from python to/from C++ into a c or c++ wrapper file and that wrapper file, and other magic is compiled into the _xxxx.so file.  This magic compilation happens via setup.py.  So when someone does import xxxx, it eventually loads the _xxxx.so shared library.

However, cmake has a way to build the needed .so file, but I don't think it uses setup.py anymore.  Just to test the theory, I started with a fresh tree, and removed lang/python/setup.py and bench/workgen/setup.py, and the build worked, and I'm able to run python and workgen.  I wouldn't remove lang/python/setup.py just yet, since there's some other magic that is involved in releasing a python version of the product, but bench/workgen/setup.py looks like it can be removed.

Reply:
I'll put together a different ticket for this.  I think you can either remove bench/workgen/setup.py in this PR, or modify it as you have here, or not, it won't matter in the end.

Reply:
WT-10480